examples/lambda/function.yaml

```
@@ -36,10 +43,97 @@ spec:
           {
             "Action": "sts:AssumeRole",
             "Principal": {
-              "Service": "lambda.amazonaws.com"
+              "Service": ["ec2.amazonaws.com", "lambda.amazonaws.com"]
             },
             "Effect": "Allow",
             "Sid": ""
           }
         ]
-      }
\ No newline at end of file
+      }
+---
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Policy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: lambda/v1beta1/function
+  labels:
+    testing.upbound.io/example-name: policy
+  name: sample-user-policy
+spec:
+  forProvider:
+    policy: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Action": "ec2:CreateNetworkInterface",
```

```suggestion
            "Action": [
                "ec2:DescribeVpcEndpoints",
                "ec2:DescribeRouteTables",
                "ec2:CreateNetworkInterface",
                "ec2:DeleteNetworkInterface",
                "ec2:DescribeSecurityGroups",
                "ec2:DescribeSubnets",
                "ec2:DescribeVpcAttribute",
                "ec2:DescribeNetworkInterfaces"
            ],
```

Could you please add these actions?